### ISSUE TITLE
Activity indicator not moves with item on swipe

### ISSUE THREAD

**Issue by LowKostKustomz** [alexaubry/BulletinBoard#38]
### New Issue Checklist

- [x] I updated BulletinBoard to the latest version.
- [x] I read the [Contribution Guidelines](https://github.com/alexaubry/BulletinBoard/blob/master/CONTRIBUTING.md).
- [x] I read the [documentation](https://alexaubry.github.io/BulletinBoard).
- [x] I searched for [existing GitHub issues](https://github.com/alexaubry/BulletinBoard/issues).

### Issue Description

When `activityIndicator` is animating and you are swiping an item, the indicator stays under the `activeSnapshotView`. When item is dismissed `activityIndicator` is still visible and disappears only after a `BulletinViewController` is dismissed. 
Also `activityIndicator`'s `isUserInteractionEnabled` property is set to `true`, so it cancels `UIPanGestureRecognizer`'s events. The events can be recognized only in `activityIndicator`'s margins area.
### Environment

- **iOS Version**: iOS 11.1.2
- **Device(s)**: iPhone 6s

**Comment by alexaubry** [comment]
That's actually the intended behavior. The item should not be dismissed if there's a task in progress. 

I fixed the bug that allowed the card to be dismissed if you swiped from the borders, but I can't think of a use case where it's useful to be able to swipe off a card when a task is in progress. Could you tell me why you'd want to do this?

**Comment by LowKostKustomz** [comment]
In my app I've Two-Factor Authentication and to enable it I want to make some requests. But during that requests user can change mind and swipe off Bulletin if he doesn't want TFA anymore . This behavior needed to not force user to wait until requests are completed.

Another example: I want to programmatically dismiss Bulletin just after a request completion and not to show success item (success will be shown with another UI elements).

And another one, not about dismissing, but maybe even more important than others: in the version I was fixing there was an issue than even if Bulletin cannot be dismissed and loading indicator is showing, you can swipe item, but loading indicator won't follow item, it will stay on the same place, which IMHO is not the best behavior.

So I think that at least last example gives the reason to fix the issue.

**Comment by alexaubry** [comment]
You're right, it's definitely better to allow the programmer to decide the user experience for their apps. Fixing this before the next version ships.
